.env.example

```
@@ -15,7 +15,13 @@ OLLAMA_PORT=""
 GEMINI_API_KEY=""
 USE_GEMINI="false"
 
-# Ports Mapping 
+# TwelveLabs Option (optional, opt-in)
+# Enables the TwelveLabs analyzer plugin: Pegasus whole-video descriptions
+# + Marengo 512-dim multimodal embeddings. Leave blank to keep things local.
+# Free API key: https://twelvelabs.io
+TWELVELABS_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)